Description: Skip test that fails with py38 until it is fixed upstream.
Author: Corey Bryant <corey.bryant@canonical.com>
Ubuntu-Bug: https://bugs.launchpad.net/bugs/1856848

--- a/keystone/tests/unit/test_hacking_checks.py
+++ b/keystone/tests/unit/test_hacking_checks.py
@@ -10,6 +10,8 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+import sys
+import testtools
 import textwrap
 
 import pycodestyle
@@ -57,6 +59,7 @@
     def get_checker(self):
         return checks.CheckForMutableDefaultArgs
 
+    @testtools.skipIf(sys.version_info >= (3, 8), "Skipping test for Python 3.8")
     def test(self):
         code = self.code_ex.mutable_default_args['code']
         errors = self.code_ex.mutable_default_args['expected_errors']
